src/shogun/lib/SGVector.cpp

```diff
@@ -518,7 +517,7 @@ template <>
 void SGVector<complex128_t>::display_vector(const complex128_t* vector, int32_t n,
 		const char* name, const char* prefix)
 {
-	ASSERT(n>=0)
+	REQUIRE(n>=0, "Vector size can not be negative.\n");
```

Comment:
see above, please display index, and it is spelled "cannot", but it should really be "must not"